harness/determined/cli/notebook.py

```diff
@@ -28,7 +28,11 @@ def start_notebook(args: Namespace) -> None:
         return
 
     nb = resp.notebook
-    maxCurrentSlotsExceeded = True if resp.maxCurrentSlotsExceeded else False
+    warnings = resp.warnings
+
+    maxCurrentSlotsExceeded = (
+        warnings and v1LaunchWarning.LAUNCH_WARNING_MAX_CURRENT_SLOTS_EXCEEDED in warnings
```

Comment:
max current slots is a misleading way to coin this, i think just "CURRENT_SLOTS_EXCEEDED" does a better job implying this may be temporary.